src/Navigation/NavigationHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled, { withTheme, DefaultTheme } from 'styled-components';
+import { Heading as H, HeadingProps } from '../Text';
+import { Mixins } from '../Utils';
+
+interface _NavigationHeaderProps extends HeadingProps {
+    label?: string;
+    theme: DefaultTheme;
+}
+
+export interface NavigationHeaderProps extends HeadingProps {
+    label?: string;
+}
+
+const _NavigationHeader: React.FC<_NavigationHeaderProps> = ({
```

Comment:
Same as before but now with '_' in the front.

Reply:
this is for private variable...